### ISSUE TITLE
How to evaluate a "good" forecast

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/evaluate-delta-for-forecasting#10]
Write up in the draft/outline is based on the work done for Germany/Poland by Nikos and me. It involves a lot of summary measures and was hard to write about + still has issues due to summarising scores across very different scales. 

Here as we are interested in transitions from big to small to big it may not make sense to summarise scores at all and instead do everything by forecast date. This might make summarising results quite hard and so having some kind of scale-free (or scale that makes sense) measure would potentially be a good idea. 

Some stan tangental links etc: https://avehtari.github.io/modelselection/CV-FAQ.html

This link should only be closed once further issues for each evaluation task have been opened.

**Comment by sbfnk** [comment]
I agree. The main hypothesis is that accounting for variants improves overall forecast performance vs. not doing so, right?

**Comment by seabbs** [comment]
I've been looking at modelling `two strain model` ~ `one strain model` * exp(coefficients)` as a way of stepping away from visual analysis (where coefficients are features like the proportion of delta variant and country). Are there theoretical issues with this?
